README.md

```diff
@@ -474,6 +474,49 @@ should now look like this:
 ![app-er-diagram-with-item_tags](https://user-images.githubusercontent.com/194400/68804955-c62eca80-065a-11ea-888b-9391154aceda.png)
 
 
+### Authentication
+
+With the addition of Sign in with Google
+using [**`elixir-auth-google`**](https://github.com/dwyl/elixir-auth-google),
+a **`sessions`** table was added to store session data
+and the **`picture`** and **`locale`** fields were added to the **`person`**:
+
+#### Create Sessions Table
+
+```
+mix phx.gen.html Ctx Session sessions auth_token:binary refresh_token:binary key_id:int person_id:references:people
```

Comment:
This command will create the schema for the session table but it will also create the html view for CRUD operations on the sessions which I don't think we will need.